support/cas-server-support-redis-ticket-registry/src/main/java/org/apereo/cas/ticket/registry/RedisTicketRegistry.java

```diff
@@ -110,6 +119,27 @@ public Ticket updateTicket(final Ticket ticket) {
         return null;
     }
 
+    private Stream<String> getKeysStream() {
+        try (Cursor<byte[]> cursor =
+                     client
+                             .getConnectionFactory()
+                             .getConnection()
+                             .scan(ScanOptions
+                                     .scanOptions()
+                                     .match(getPatternTicketRedisKey())
+                                     .count(SCAN_COUNT)
+                                     .build())) {
+            return StreamSupport
+                    .stream(Spliterators.spliteratorUnknownSize(cursor, Spliterator.ORDERED), false)
+                    .map((key) -> (String) client.getKeySerializer().deserialize(key))
+                    .collect(Collectors.toList())
```

Comment:
I'd personally prefer to make a proper stream without materializing it to `List`. But we should make this stream closeable to properly close the `Cursor`. If we do it we should wrap `getKeysStream` and `getKeys(Predicate)` to try-with-resources everywhere. I can do it if it is OK.

Reply:
Sure, sounds OK to me.

Also, please make sure you're using the correct indentations. (Looks like you're using tabs in some places)

Reply:
I did not find any tabs in my code.